docs/chain/collator.md

```diff
@@ -103,6 +103,14 @@ This makes sure that the keyfiles are not accidentally lost or published when th
 You can configure where to store the session keys using the `--keystore-path` option.
 Since the collator will collate only for the parachain, there is no need to add this to the relaychain part of the command.
 
+#### Storage Root Error
+
+Due to an issue in the caching implementation in substrate, it is recommended to reduce the cache size.
+Otherwise the node might get blacklisted by other peers and ultimately disconnected from the p2p network.
+This would prevent a collator from producing blocks.
```

Comment:
Eventually, this leads to being banned by the peers until the issue spreads slowing the chain down.

Reply:
like this: 45e8614